### PR TITLE
updated warranty link

### PR DESCRIPTION
**Pull request by DavidVaini**

### CODE REVIEW

File: src/data/footer.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = {
 			value: 'NEWS',
 		},
 		{
-			href: 'http://survey.curtmfg.com/surveys/5',
+			href: 'http://warranty.curtgroup.com/surveys/5',
```
Comment (dgallegos) [inline src/data/footer.js]: Can this be https or //

Comment (DavidVaini) [inline src/data/footer.js]: does the https version work for you? Mine keeps timing out.

Comment (dgallegos) [inline src/data/footer.js]: Just realized that's broken, let me check if our certificate is wildcard or not.

Comment (dgallegos) [inline src/data/footer.js]: I don't think its a wildcard, so we can do this.

**Pull request merged**